sdk/eventhubs/azure-messaging-eventhubs-track1-perf/pom.xml

```diff
@@ -25,6 +25,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Had to specify an explicit version because it resolved an old one and failed at runtime.